pkg/did/client_test.go

```diff
@@ -27,11 +27,98 @@ import (
 	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/models"
 )
 
+func TestClient_UpdateDID(t *testing.T) {
+	t.Run("test domain is empty", func(t *testing.T) {
+		v := New()
+
+		pubKey, privKey, err := ed25519.GenerateKey(rand.Reader)
+		require.NoError(t, err)
+
+		err = v.UpdateDID("did:ex:123", "", WithNextUpdatePublicKey(pubKey),
```

Comment:
- looks good, I like how it sounds: with next update key, with signing key
- make sure you include test cases where you pass in multiple public keys (ids) and multiple services (ids) at the same time

Reply:
done